camel/memories/agent_memories.py

```diff
@@ -130,7 +177,13 @@ def retrieve(self) -> List[ContextRecord]:
         vector_db_retrieve = self.vector_db_block.retrieve(
             self._current_topic, self.retrieve_limit
         )
-        return chat_history[:1] + vector_db_retrieve + chat_history[1:]
+        graph_db_retrieve = self.graph_db_block.retrieve()
```

Comment:
here will retrieve all messages stored in graph db, why not limit it to specific topic like how `vector_db_retrieve` did?